setup.py

```diff
@@ -1089,6 +1089,60 @@ def check_pydep(importname: str, module: str) -> None:
 
 
 class build_ext(setuptools.command.build_ext.build_ext):
+    def _wrap_headers_with_macro(self, include_dir: Path) -> None:
```

Comment:
@janeyx99 fyi I moved this from bdist_wheel to build_ext so that both 

`python -m pip install --no-build-isolation` and 

`python setup.py bdist_wheel`

paths will get the change